src/main/java/seedu/address/model/person/Driver.java

```diff
@@ -0,0 +1,26 @@
+package seedu.address.model.person;
+
+import seedu.address.model.tag.Tag;
+
+import java.util.Set;
+
+public class Driver extends Person {
+    private AvailabilityManager allAssignedTimeSlots;
+    private int id;
+    static int IDCOUNT = 1;
+
+    public Driver (Name name, Phone phone, Email email, Address address, Set<Tag> tags) {
+        super(name, phone, email, address, tags);
+        allAssignedTimeSlots = new AvailabilityManager();
+        id = IDCOUNT;
+        IDCOUNT++;
```

Comment:
Would `IDCOUNT` be sufficient already so we do not need assign the same value to `private int id`?